modules/default-values/assets/js/editor/hooks/ui/default-values/base-loader-hide.js

```diff
@@ -0,0 +1,9 @@
+import { After } from 'elementor-api/modules/hooks/ui';
+
+export class BaseLoaderHide extends After {
```

Comment:
Why with hooks and not an internal command?

Reply:
2 commands? one for show loader one for hiding? or 1 command that takes param? or 1 command that takes promise as param and when the promise ends it hide the loader?

Reply:
> Why with hooks and not an internal command?

Why not with hooks and do command internal?

Reply:
Can you try with the new UI states?

Reply:
Done